app/src/main/java/com/example/lhi06/residue/MainActivity.java

```diff
@@ -0,0 +1,70 @@
+package com.example.lhi06.residue;
+
+import android.os.Bundle;
+import android.support.design.widget.FloatingActionButton;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+public class MainActivity extends AppCompatActivity {
+
+    @BindView(R.id.add_review) FloatingActionButton addReviewButton;
+    @BindView(R.id.request) EditText searchField;
+    @BindView(R.id.searchButton) Button searchButton;
+    @BindView(R.id.toolbar) Toolbar toolbar;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        ButterKnife.bind(this);
+        setSupportActionBar(toolbar);
+
+        searchButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                //TODO: Code for searching for reviews done
+                //String artistName = editText1.getText().toString();
+                //handler.urlBuilder(artistName);
+            }
+        });
+
+        addReviewButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                //TODO: Navigate to the album searching activity
+
+            }
+        });
+
+    }
+
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+         //Inflate the menu; this adds items to the action bar if it is present.
+        getMenuInflater().inflate(R.menu.menu_main, menu);
+        return true;
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+         //Handle action bar item clicks here. The action bar will
+         //automatically handle clicks on the Home/Up button, so long
+         //as you specify a parent activity in AndroidManifest.xml.
+        int id = item.getItemId();
+
+         //noinspection SimplifiableIfStatement
+        if (id == R.id.action_settings) {
```

Comment:
Im not sure, it was here when i created the activity so i think im going to leave it incase i need it at any point